styles/_variables.scss

```diff
@@ -35,6 +35,7 @@ $padSm: $pad / 2;
 $padMd: $pad * 1.5;
 $padMd2: $pad * 1.2;
 $padLg: $pad * 2;
+$padLg2: $pad * 4;
```

Comment:
We should keep numbered classes in sequence, so if we need something bigger than the huge class, we should make another class after Huge. Maybe Gigantic? (padGi) Or just end at huge and do padHg2, padHg3, etc.

Generally I tried to avoid the numbered size classes, and only used them when something had to fit between two other sizes.